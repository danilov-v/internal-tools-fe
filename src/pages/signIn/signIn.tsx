import React from 'react';
import { RouteComponentProps } from '@reach/router';

import { LoginForm } from 'components/LoginForm';

export const SignIn: React.FC<RouteComponentProps> = () => {
  return (
    <div>
      <LoginForm />
    </div>
  );
};
