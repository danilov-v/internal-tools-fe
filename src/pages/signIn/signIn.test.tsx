import React from 'react';
import { render, RenderResult } from '@testing-library/react';

import { SignIn } from './signIn';

describe('pages/signIn', () => {
  it('should render and match snapshot', () => {
    const component = render(<SignIn />);

    expect(component.baseElement).toMatchSnapshot();
  })
});