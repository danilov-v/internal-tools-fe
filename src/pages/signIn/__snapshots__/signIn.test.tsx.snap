// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/signIn should render and match snapshot 1`] = `
<body>
  <div>
    <div>
      <form>
        <input
          type="text"
        />
        <br />
        <input
          type="password"
        />
        <br />
        <button
          type="submit"
        >
          Войти в ИТ
        </button>
      </form>
    </div>
  </div>
</body>
`;
